extensions/coap/CMakeLists.txt

```diff
@@ -70,19 +70,6 @@ include_directories(${COAP_INCLUDE_DIRS})
 target_link_libraries (nanofi-coap-c ${COAP_LIBRARIES})
 target_link_libraries (nanofi-coap-c ${HTTP-CURL})
 target_link_libraries (minifi-coap nanofi-coap-c ${COAP_LIBRARIES})
-if (WIN32)
```

Comment:
Some of the cruft that has been copy/pasted. Should consider stubbing out some of this into modular cmake in the future.